.github/workflows/release-tag.yml

```diff
@@ -0,0 +1,15 @@
+name: Re-tag releases
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Actions-R-Us/actions-tagger@v2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          publish_latest_tag: true
```

Comment:
Just curious — What effect does this have?

Reply:
Also keeps a `@latest` tag up to date with the latest release.